test/forms/SelectTransactionsFormProviderSpec.scala

```diff
@@ -122,7 +122,7 @@ class SelectTransactionsFormProviderSpec extends SpecBase {
     "populate CashTransactionDates form correctly (or with correct error) for input end date" when {
 
       "end date is valid and uses the last day of the month" in new SetUp {
-        val expectedEndDate = LocalDate.of(2021, 10, 31)
+        val expectedEndDate: LocalDate = LocalDate.of(year, month, day + 30)
```

Comment:
do we want to just add another value for the last day of October rather than doing a day + 30?